content/Day-7/inclass-activity/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Instructions"
+weight: 102
+---
+
+### Activity : Basic Network Troubleshooting
+This activity is designed to introduce learners to a basic guide to troubleshooting using network commands. Learners should open the command prompt or terminal and enter the commands listed below. These commands have to use should be appended to the terms from the reading "Network Terms" 
```

Comment:
These commands have to use should be appended to the terms from the reading "Network Terms" - clarity on this sentence